src/main/java/picard/analysis/CollectWgsMetricsWithNonZeroCoverage.java

```diff
@@ -109,33 +110,30 @@ public static void main(final String[] args) {
     @Override
     protected int doWork() {
         IOUtil.assertFileIsWritable(CHART_OUTPUT);
+        return super.doWork();
+
```

Comment:
The plotting works in Nils's latest branch so he must have fixed it. Will uncomment.